custom/MCChassisController.java

```diff
@@ -8,15 +8,15 @@
 import org.usfirst.frc4904.standard.custom.sensors.InvalidSensorException;
 import edu.wpi.first.wpilibj.PIDSourceType;
 
-public class PIDChassisController implements ChassisController {
+public class MCChassisController implements ChassisController {
 	protected ChassisController controller;
 	protected double maxDegreesPerSecond;
 	protected double targetYaw;
 	protected double lastUpdate;
 	protected IMU imu;
 	protected MotionController motionController;
 	
-	public PIDChassisController(ChassisController controller, IMU imu, MotionController motionController, double maxDegreesPerSecond) {
+	public MCChassisController(ChassisController controller, IMU imu, MotionController motionController, double maxDegreesPerSecond) {
```

Comment:
The official pronunciation of this class is "mick chassis controller" as if it were a McDonalds product.